_mobile/ios.md

```diff
@@ -192,7 +191,38 @@ In the build settings, search for **other linker flags**.  Add a custom linker f
 
 ## API Docs
 
-Currently, the iOS framework uses the Pytorch C++ front-end APIs directly. The C++ document can be found here https://pytorch.org/cppdocs/. To learn more about it, we recommend exploring the [C++ front-end tutorials](https://pytorch.org/tutorials/advanced/cpp_frontend.html) on PyTorch webpage. In the meantime, we're working on providing the Swift/Objective-C API wrappers to PyTorch.
+Currently, the iOS framework uses the Pytorch C++ front-end APIs directly. The C++ document can be found here [https://pytorch.org/cppdocs/](https://pytorch.org/cppdocs/). To learn more about it, we recommend exploring the [C++ front-end tutorials](https://pytorch.org/tutorials/advanced/cpp_frontend.html) on PyTorch webpage. In the meantime, we're working on providing the Swift/Objective-C API wrappers to PyTorch.
+
+
+### Custom Build
+
+Starting from 1.4.0, PyTorch supports custom build. You can now build the PyTorch library that only contains the operators needed by your model. To do that, follow the steps below
+
+1) Verify your PyTorch version is 1.4.0 or above. You can do that by checking the value of `torch.__version__`.
```

Comment:
You can also use the syntax 1\. to keep the numbering left-aligned. This will match how it looks in Android: 
![image](https://user-images.githubusercontent.com/8042156/71197691-0a5b4f00-2247-11ea-94c3-fe9ff649112a.png)